.gitignore

```diff
@@ -102,6 +103,15 @@ node_modules
 testApp
 coverage
 
+############################
+# Meta Tests
+############################
+
+src/api/code-challenge/content-types/code-challenge/curriculum-helpers.js
+src/api/code-challenge/content-types/code-challenge/general.js
+src/api/code-challenge/content-types/code-challenge/test-evaluator.js
+src/api/code-challenge/content-types/code-challenge/utils.js
```

Comment:
This is not a scaleable way of doing things as you have to add a new entry whenever you have a new file. Try using a glob pattern instead